Shreya/HTML Basics Remake/form.html

```diff
@@ -8,22 +9,22 @@
         <h1>Register</h1>
             <p>
                 <label for="first">First Name</label>
-                <input type="text" name="first" placeholder="First Name" required>
+                <input type="text" id="first" placeholder="First Name" required>
                 <label for="last">Last Name</label>
-                <input type="text" name="last" placeholder="Last Name" required>
+                <input type="text" id="last" placeholder="Last Name" required>
             </p>
             <p>
-                <label for="gender">Gender:</label>
-              <label for="gender">Male:</label>
-                <input type="radio" name="gender" required>
-                <label for="gender">Female:</label>
-                <input type="radio" name="gender" required>
-                <label for="gender">Others:</label>
-                <input type="radio" name="gender" required>
+                <label for="1">Gender:</label>
```

Comment:
Label not required here.

Reply:
then what tag shall I wrap Gender in ?

Reply:
> then what tag shall I wrap Gender in ?

Refer this: [What is the HTML for=“” attribute in \<label>? - Stackoverflow](https://stackoverflow.com/a/45307833/7023294)

Reply:
Still incomplete